SAP/Script/FXVProduct.py

```diff
@@ -120,29 +119,17 @@ def sap_test():
     if(NameMapping.Sys.browser.pageFlp.frameApplicationSalesdocumentCre.WaitNamedChild("panelContinue", 75000).Exists):
       textbox.FindElement("//div[.='Continue']").Click()
     page.WaitConfirm(3000)
-  
     image = page.FindElement("//header[contains(@class, 'sapUshellShellHeader')]")
     image.FindElement("//a[@title='Navigate to Home Page']").Click()
+    
+    #SearchBOX
     searchBox.search_item(page, "csk2")
-    section = browser.pageFlp.sectionShellSplitCanvas
-    section.FindElement("//span/span/span[contains(text(), 'Multi-')]").Click()
-    item_field = textbox.FindElement("//input[@id=(//label[.='Item']/@for)]")
-    item_field.SetText("100")
-    material_field = textbox.FindElement("//input[@id=(//label[.='Material']/@for)]")
-    material_field.SetText("FXV")
-    textbox2 = textbox.FindElement("//input[@id=(//label[.='BOM Application']/@for)]")
-    textbox2.SetText("PP01")
-    textbox.FindElement("//div[.='Execute']").Click()
     
-    frame = panel.frameApplicationBillofmaterialMu
-    form = frame.formWebguiform0
-    if(NameMapping.Sys.browser.pageFlp.frameApplicationBillofmaterialMu.WaitNamedChild("panelSpreadsheetCtrlShiftF7", 10000).Exists):
-     form.FindElement("//div[@title='Spreadsheet... (Ctrl+Shift+F7)']").Click()
-    if(NameMapping.Sys.browser.pageFlp.frameApplicationBillofmaterialMu.WaitNamedChild("panelUpdowndialogchoose", 10000).Exists):
-      frame.FindElement("//div[.='OK']").Click()
+    #Multi-Level Sales Order BOM
+    MultiLevelBOM.multi_level_bom(panel,textbox,page,standard_order_values(sap_field_values))
```

Comment:
Use direct BOM material value.

Reply:
removed the pervious parameter and pass the direct  BOM material's Json value